src/app/app-routing.module.ts

```diff
@@ -1,11 +1,17 @@
 import { NgModule } from '@angular/core';
 import { Routes, RouterModule } from '@angular/router';
 
+import { trimEnd } from 'lodash';
+
 import { LoginComponent } from './login/login.component';
 import { SigninComponent } from './signin/signin.component';
 import { ContextResolver } from './shared/context-resolver.service';
 
 
+export function removeAction(url: string) {
```

Comment:
Yes, wasn't sure where to put it. It's a hack really, maybe angular 4 will fix it.

We can make it a pipe, but I'm not sure how you call a pipe from within a component?